docs/v0.5/runtimes/streaming.md

```diff
@@ -6,22 +6,36 @@ sidebar_label: Streaming
 
 The streaming runtime allows execution of functions on **streams** of messages, permitting more complex interactions than the simple request / reply used by the [core](core.md) or [knative](knative.md) runtimes.
 
+> NOTE: Not all invokers support streaming. Invokers conforming to the full invoker [specification](https://github.com/projectriff/invoker-specification) can be used with the streaming runtimes, while some can't. In particular, the [command](../invokers/command.md) invoker does not support streaming.
+
 ## Install
 
 The Streaming runtime is not installed with riff by default. See the [getting started guide](../getting-started.md) for how to install the prerequisits and riff Build in your Kubernetes environment.
 
 You can then install the Streaming runtime and it's dependencies using the following:
 
+### install KEDA
+
+> NOTE: For Kubernetes versions 1.16 and later the use of `k8s.io` in the KEDA CRDs results in a [failed status condition](https://github.com/kedacore/keda/issues/552) and we need to avoid the kapp reconciliation for the KEDA resources.
```

Comment:
rm let's fix this in the install

Reply:
what do you mean by "fix this in the install"?

Reply:
https://github.com/projectriff/release/pull/129